src/style/global.sass

```diff
@@ -31,3 +32,50 @@ ul, ol
     padding-inline: 2rem
     width: 100%
     box-sizing: border-box
+
+.centered-container 
+    display:flex
+    width: 100%
+    justify-content:center
+    align-items:center
+    
+/* classes for buttons (jonathan) */
```

Comment:
All classes for buttons should be moved to a separate file, under `Button` component.